components/ContentCard.stories.js

```diff
@@ -0,0 +1,15 @@
+import { storiesOf } from '@storybook/vue';
+
+import ContentCard from './ContentCard.vue';
+
+storiesOf('ContentCard', module)
+  .add('card', () => ({
+    components: { ContentCard },
+    template:  `<b-col cols="3">
+    	<ContentCard 
+          cardTitle="This is a Storybook Card"
+          contentSource="card"
+          imageUrl="https://api.europeana.eu/api/v2/thumbnail-by-url.json?size=w400&type=IMAGE&uri=http%3A%2F%2Fwww.epaveldas.lt%2FrecordObject%2FLDM%2F1_1013608" 
```

Comment:
rather than hotlinking to 'our' image, let's add an image to ./.storybook/assets/img/portatrait.jpeg, then change the -s param in the sotrybook scripts section of package.json to "-s ./.storybook/assets,./" and load the image here via           imageUrl="/img/sketch-portrait.jpeg" 

That way we are not dependant on external resources when displaying the sotrybook and since storybook specific assets are only in the .storybook directory they won't be included when running generate and we can push them specifically when we deploy the storybook.